src/app/components/sidenav/sidenav.component.scss

```diff
@@ -22,7 +21,7 @@
 }
 
 .icon-description{
-    height: 45px;
+    height: 49.9999px;
```

Comment:
why do you need to use such a value? :)

Reply:
I wanted to line it up perfectly, but I think I found a prettier value that gets probably as close as this one.